src/timer/CountDownTimer.test.jsx

```diff
@@ -29,15 +29,15 @@ describe('ExamTimerBlock', () => {
           attempt_status: 'started',
           exam_url_path: 'exam_url_path',
           exam_display_name: 'exam name',
-          time_remaining_seconds: 10,
-          low_threshold_sec: 15,
-          critically_low_threshold_sec: 5,
+          time_remaining_seconds: 24,
```

Comment:
yeah, the previous test values we all in seconds but allotted time is in minutes (converted to seconds). The previous test values are impossible to produce because the theoretical allotted time to create them was 75 seconds.